todo_app/app.py

```diff
@@ -1,4 +1,11 @@
+from werkzeug.utils import redirect
+from todo_app.data.session_items import delete_item, get_items
+from todo_app.data.session_items import add_item
+from todo_app.data.session_items import get_item
+from todo_app.data.session_items import save_item
 from flask import Flask
+from flask import render_template
+from flask import request
```

Comment:
You can tidy these imports up

```suggestion
from flask import Flask, render_template, request, redirect
from todo_app.data.session_items import delete_item, get_items, save_item, add_item, get_item
```

Reply:
Is there any way to import all the functions from session.items?
If you have 2 functions with the same name, how would you disambigufy (new word) them?

Reply:
To import everything from a module use `*`, so
```python
from todo_app.data.session_items import *
```
You can rename things with `as`, so if you did want both redirect functions you could do
```python
from werkzeug.utils import redirect as werkzeugRedirect
from flask import redirect as flaskRedirect
```